juju/paths/paths.go

```diff
@@ -28,10 +28,18 @@ const (
 	cloudInitCfgDir
 )
 
+const (
+	// NixDataDir is location for agent binaries on *nix operating systems.
+	NixDataDir = "/var/lib/juju"
+
+	// NixDataDir is location for Juju logs on *nix operating systems.
```

Comment:
This is simply a standard `*nix` log dir. You must append "/juju/" to this to get the location for Juju logs on *nix operating systems. Some areas of the code are already doing just that.

Reply:
I find it confusing that you have the NixDataDir have the juju suffix but not the NixLogDir (which has a copy / paste error in the comment BTW.

Reply:
I agree, that is very confusing. I coded this in such a way that it will require minimal change to forward port (for whatever that is worth). The copy Pasta was intentional, when this gets forward ported I will change it will change this and a few other things that are incorrect with respect to the changes in this PR.